samwise.js

```diff
@@ -6,30 +6,32 @@
  */
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.